packages/minifier-js/minifier.js

```diff
@@ -47,3 +47,37 @@ export const meteorJsMinify = function (source) {
 
   return result;
 };
+
+
+
+export const meteorJsMinifyAsync = async function (source) {
+  const NODE_ENV = process.env.NODE_ENV || "development";
+
+
+  const options = {
+    compress: {
+      drop_debugger: false,  // remove debugger; statements
+      unused: false,         // drop unreferenced functions and variables
+      dead_code: true,       // remove unreachable code
+      typeofs: false,        // set to false due to known issues in IE10
+      global_defs: {
+        "process.env.NODE_ENV": NODE_ENV
+      }
+    },
+    // Fix issue #9866, as explained in this comment:
+    // https://github.com/mishoo/UglifyJS2/issues/1753#issuecomment-324814782
+    // And fix terser issue #117: https://github.com/terser-js/terser/issues/117
+    safari10: true,          // set this option to true to work around the Safari 10/11 await bug
+  };
+
+  try {
+    const terserResult = await terserMinify(source, options);
+    const response = {
+      code: terserResult.code,
+      minifier: "terser"
+    };
+    return response;
+  } catch (e) {
+    throw e;
```

Comment:
There's no need to `try` if you're rethrowing the same error.

Reply:
Inline approach will solve this.